operations/jsonnet/scalable-single-binary/config.libsonnet

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
i'm concerned about adding more untested jsonnet. we will not keep this up to date as Tempo moves forward.

How do we have good deployment examples like this that we can also make sure remain maintained? If they are not maintained they will end up doing more harm then good.

Reply:
I'm currently running this in a lab environment, so I know that it does work.  There will be a fast follow PR to include a headless service for member joining.  I can split this out into a different PR if preferred, or if we don't care to maintain another deployment strategy that's fine too.  We also have some Helm to think about if we want to keep all ecosystems updated with all deployment scenarios.

Reply:
I've dropped the jsonnet.